### DISCUSSION TITLE
Git Hub Pages

### DISCUSSION THREAD

**Discussion by Saywlee** [community/community#56239] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/community/community/assets/10578872/2204c4b8-a473-47bd-b32c-e3192b6f7dd3)
I created a repository and clicked on Settings but I couldn't find the Git Hub Pages. So I could not use the Automatic page generator. Please help. Thanks.

**Comment by ssambender** [comment]
In your repository settings, the Github Pages will be on the left side and you can change settings in there.
![image](https://github.com/community/community/assets/80929018/2373d4a8-ab15-4ced-932a-812ce05ef5fc)
You may have to change from which branch the page is set to, likely to main.

Or you can set up your pages directly through the [GitHub Pages website](https://pages.github.com/).
